src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Waiting/IWaitContext.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Waiting
+{
+    /// <summary>
+    /// A dispose-able context object that considers the operation on which it is
+    /// waiting to be complete once dispose it called.
+    /// </summary>
+    internal interface IWaitContext : IDisposable
```

Comment:
Why does this interface exist? It's not used outside of the implementation of VisualStudioOperationWaitIndicator.